src/structures/contexts/DiscordSlashMessageContext.ts

```diff
@@ -9,21 +9,15 @@ import MessageContext from "./MessageContext.js";
 type M = Promise<Message<boolean>>;
 
 export default class DiscordSlashMessageContext extends MessageContext {
-	private interaction: CommandInteraction;
-	public author: User;
 	public command: Command;
-	public guild?: Guild;
 
 	constructor(
 		client: Client,
```

Comment:
`client`'s access specifier should be explicited.
```suggestion
		public client: Client,
```